### PR TITLE
Revert "Use Laboratory to test Formplayer Queries"

### PR DESCRIPTION
**Pull request by czue**
Reverts dimagi/touchforms#204

this breaks the build until https://github.com/dimagi/commcare-hq/pull/10629 is merged. seems safer to revert. @benrudolph @wpride

### CODE REVIEW

File: touchforms/formplayer/tests/test_basic.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import os
+from django.test import TestCase
+from touchforms.formplayer.models import XForm
+from django.conf import settings
```
Comment (dimagimon) [inline touchforms/formplayer/tests/test_basic.py]: F401 'settings' imported but unused

File: touchforms/formplayer/api.py [hunk 2]
```diff
@@ -250,19 +249,7 @@ def server_down(cls):
...
-    up = urlparse(url)
-    headers = {}
-    headers["content-type"] = content_type
-    headers["content-length"] = len(data)
-    conn = httplib.HTTPConnection(up.netloc)
-    conn.request('POST', up.path, data, headers)
-    resp = conn.getresponse()
-    results = resp.read()
-    return results
+
```
Comment (dimagimon) [inline touchforms/formplayer/api.py]: W293 blank line contains whitespace

**Comment by benrudolph** [comment]
thanks cory

**Comment by wpride** [comment]
hold off on merge - if this passes https://github.com/dimagi/commcare-hq/pull/10629 and is merged then we won't need to revert

**Comment by czue** [comment]
@wpride assume you'll close this if not relevant

**Pull request closed without merging**